src/arch/ppc64/include/arch/io.h

```diff
@@ -5,7 +5,8 @@
 
 #include <stdint.h>
 
-#define MMIO_GROUP0_CHIP0_LPC_BASE_ADDR 0x0006030000000000
+/* Add 1 to ignore HRMOR */
+#define MMIO_GROUP0_CHIP0_LPC_BASE_ADDR 0x0006030000000001
```

Comment:
This will break the access to given IO port. IO ports can be typically accessed at byte granularity